test/containerized_test.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2022 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+
+import json
+import http.client as http_client
+
+import pytest
+
+from ovirt_imageio._internal import config
+from ovirt_imageio._internal import server
+from ovirt_imageio._internal.units import KiB
+
+from . import testutil
+from . import http
+
+
+TICKET_IMG_UUID = "file"
+TICKET_IMG_SIZE = 16 * KiB
+
+
+@pytest.fixture(scope="module")
+def tmp_image(tmpdir_factory):
+    img_path = tmpdir_factory.getbasetemp() / "disk.raw"
```

Comment:
Please use the style of the current code for creating temporary directories. Since we don't really need module scope, we can simply use the `tmpdir` fixture.

Reply:
I used it because it was used by the `srv` ficture that had the module scope.